joomla/joomla-cms
# [4.0] Destroy session on deleting installation folder

Destroy the session after installation is finished

### Summary of Changes
Remove the session


### Testing Instructions
Install joomla 4.0 and check if the sessions file still exists in the php session directory.


### Expected result
No session data.
